grunttasks/serve.js

```diff
@@ -5,9 +5,15 @@
 module.exports = function (grunt) {
   'use strict';
 
-  grunt.registerTask('serve', [
-    'build:development',
-    'hapi',
-    'watch'
-  ]);
+  grunt.registerTask('serve', 'Builds the dist assets, starts the Hapi server, and watches for changes.', function (target) {
+    if (!target) {
+      target = 'development';
+    }
```

Comment:
I don't understand where `target` is getting set, but it seems nice to use `config.get('env')` if we can. Rather than handle all four environments differently, could you just, say, do one thing `if (config.get('env') === 'local')`, and do something else for all other cases?